### DISCUSSION TITLE
Is the SDK for frontend only?

### DISCUSSION THREAD

**Discussion by akashvekariya** [microsoft/omnichannel-chat-sdk#255] [Q&A] [answered]
Hello, 

I want to know if this SDK is only for frontend? as everything in document is suggesting in tsx even samples. My requirement is to make the rest api's to work with other chat UI. I want to make specific endpoints.

Let's say If I hit /handover it triggers initialize and startChat methods of SDK. Is it possible to build this in node.js?

**Comment by xTEddie** [comment, marked as answer]
Yes, `omnichannel-chat-sdk` is taking dependencies on different `Web` libraries. So, it can only be run as frontend.

**Comment by alexsam986** [comment]
No, I think you misunderstood the concept of chat SDKs. It is versatile and can handle your whole chat infrastructure from front end to backend. 

Some of the basic capabilities of SDKs include integration with existing websites or applications and enabling chat, voice, and video capabilities. 
Moreover, with custom self-hosted chat SDKs, it's even more powerful for you to customize the features, brandings, and even more advanced features too. 

If you want to know more about the best SDKs in the industry, check out the blog here: [https://www.contus.com/blog/best-chat-sdk/](https://www.contus.com/blog/best-chat-sdk/)
